### ISSUE TITLE
New cop for alignment of arguments on repeated, consecutive method calls

### ISSUE THREAD

**Issue by JasonBarnabe** [rubocop/rubocop#9242]
When a method is repeated multiple times consecutively (common in a DSL), ensure the arguments are aligned.

Good:
```ruby
has_many :invoices,  dependent: :destroy
has_many :orders,    dependent: :nullify, inverse_of: :user
has_many :purchases, dependent: :nullify, inverse_of: :user
```

Bad:
```ruby
has_many :invoices, dependent: :destroy
has_many :orders, dependent: :nullify, inverse_of: :user
has_many :purchases, dependent: :nullify, inverse_of: :user
```

Bad:
```ruby
has_many :invoices,  dependent: :destroy
has_many :orders,    dependent: :nullify, inverse_of: :user
has_many :purchases, inverse_of: :user,   dependent: :nullify
```

**Comment by bbatsov** [comment]
Closing as stale — only 1 reaction, no comments in 5+ years. If someone wants this cop, please open a fresh issue with a concrete proposal.
